core/src/themes/ionic/ionic.mixins.scss

```diff
@@ -41,6 +41,22 @@
   pointer-events: none;
 }
 
+// Focused mixin to be used on regular elements for the Ionic Theme focused state
+//
+// ex: :host(.ion-focused) .toggle-icon {
+//       @include globals.focused-state();
+//      }
+//
+// --------------------------------------------------
+@mixin focused-state(
```

Comment:
Would it be less confusing to keep the arguments in the same order as `outline` and also make offset optional / a boolean?

Reply:
https://github.com/ionic-team/ionic-framework/pull/29805/commits/ebbb8cb29f754baa4902bf2a70185b67a4ede06a